### PR TITLE
New tests for iframes functionality

### PR DESCRIPTION
**Pull request by gallyamb**
* Use mocks where possible
 * New tests for FrameSwitcher
 * Remove unusued/redundant files

### CODE REVIEW

File: player/src/test/groovy/com/focusit/player/specs/SeleniumDriverSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    JSONObject getSimpleEvent() {
+        JSONObject event = new JSONObject()
+        event.put("tabuuid", "2")
+        event.put("window.width", 1500)
+        event.put("window.height", 1500)
+        return event
+    }
+
+    WebDriver getWd(JSONObject event) {
+        return Mock(RemoteWebDriver);
```
Comment (d0k1) [inline player/src/test/groovy/com/focusit/player/specs/SeleniumDriverSpec.groovy:120]: What is the reason to use mocks there instead real browser?

File: player/src/test/groovy/com/focusit/player/specs/FrameSwitcherSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * Created by Gallyam Biktashev on 16.02.17.
+ */
+class FrameSwitcherSpec extends BaseSpec {
+    WebDriver webDriver
+    WebDriver.TargetLocator locator
+    String windowHandle
+    JSONObject event
+
+    def setup() {
+        webDriver = Mock()
```
Comment (d0k1) [inline player/src/test/groovy/com/focusit/player/specs/FrameSwitcherSpec.groovy:18]: there could be a real browser.
Test would be more than real in this case :)

File: player/src/test/groovy/com/focusit/player/specs/SeleniumDriverSpec.groovy (hunk 1 above)
Comment (gallyamb) [reply player/src/test/groovy/com/focusit/player/specs/SeleniumDriverSpec.groovy:120]: Well, we want to test the particular part of our application, and, therefore, we don't need to setup all of the environment. The testing of the real web driver isn't our care. This is the unit tests and, imo, they should be as simple and fast as possible.

File: player/src/test/groovy/com/focusit/player/specs/FrameSwitcherSpec.groovy (hunk 2 above)
Comment (gallyamb) [reply player/src/test/groovy/com/focusit/player/specs/FrameSwitcherSpec.groovy:18]: The same as above

**Pull request merged**